.github/workflows/build-staging-ios.yml

```diff
@@ -60,7 +69,7 @@ jobs:
           GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
           MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
       - name: Replace ipa bundle
-        if: steps.ipa-cache.outputs.cache-hit == 'true'
+        if: inputs.force-build == 'true' || steps.ipa-cache.outputs.cache-hit == 'true'
```

Comment:
The other changes are logical to me, but what does `inputs.force-build == 'true' || ...` implies here?

Reply:
It should run it in any case but then I looked at another build, for example, the cache was changed and this is skipped, so I changed the logic a bit to fit.